packages/amplify_core/lib/src/types/models/auth_rule.dart

```diff
@@ -84,24 +84,8 @@ class AuthRule {
       ..removeWhere((k, dynamic v) => v == null);
   }
 
-  factory AuthRule.fromMap(Map<String, dynamic> map) {
```

Comment:
Did the linter suggest to remove these two factories as they are not used or something?

Reply:
They aren't being used in the codebase and threw a bunch of lint errors. I started fixing them but realized some were just incorrect so I gave up and just removed em.